src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -148,12 +142,21 @@
 </div>
 </div>
 <footer>
-    <span>Gradle Profiler version {{ environment.profilerVersion }}</span>
+    <span>Gradle Profiler version {{ benchmarkResult.environment.profilerVersion }}</span>
 </footer>
 </div>
 <script>
 Array.prototype.unique = function() { return [...new Set(this)]; };
 
+// Initialize sample config
+benchmarkResult.scenarios.forEach((scenario, scenarioIndex, scenarios) => {
+    scenario.samples.forEach((sample, sampleIndex, samples) => {
+        sample.color = `hsl(${scenarioIndex * 360 / scenarios.length}, ${100 - 80 * sampleIndex / samples.length}%, ${30 + 40 * sampleIndex / samples.length}%)`;
```

Comment:
I find the shades of red/green very hard to distinguish: 
![image](https://user-images.githubusercontent.com/423186/93098430-1fb2bc00-f6a7-11ea-969c-f899e8809493.png)

Do you have any better ideas what colors to use?

Reply:
Not sure how important it is to distinguish the different shades of green (=different build operations for the same experiment), it seems more important to distinguish the green from the red (same build operations for different experiments).

Reply:
That said I'm open to suggestions.